pkg/query-service/model/logs.go

```diff
@@ -49,7 +49,8 @@ func GetLogFieldsV3(ctx context.Context, queryRangeParams *v3.QueryRangeParamsV3
 				if pass {
 					continue
 				}
-				data[selectedField.Name] = v3.AttributeKey{
+				name := selectedField.Name + "##" + fieldType.String() + "##" + selectedField.DataType
+				data[name] = v3.AttributeKey{
```

Comment:
The change in the key format needs to be handled wherever `GetLogFieldsV3` is used. A quick search says it's used to create links. Please check.

Reply:
oh right, while all others are using it for enrichment, `metaForLinks` is using it differently.

Reply:
Thanks have made the changes for this. I didn't past the alertType as passing it results in cyclic dependency so went ahead with a boolean flag.